lib/mayaUsd/fileio/chaser/exportChaserRegistry.cpp

```diff
@@ -51,12 +51,15 @@ TF_INSTANTIATE_SINGLETON(UsdMayaExportChaserRegistry);
 
 std::map<std::string, UsdMayaExportChaserRegistry::FactoryFn> _factoryRegistry;
 
-bool UsdMayaExportChaserRegistry::RegisterFactory(const std::string& name, FactoryFn fn)
```

Comment:
Same parameter fromPython added at every Registration to determine if registration is done in c++ (within a .dll) or from Python.